docs/integrations/sources/intercom.md

```diff
@@ -41,11 +40,13 @@ The connector is restricted by normal Intercom [requests limitation](https://dev
 
 The Intercom connector should not run into Intercom API limitations under normal usage. Please [create an issue](https://github.com/airbytehq/airbyte/issues) if you see any rate limit issues that are not automatically retried successfully.
 
-## Getting started
+## Getting Started (Airbyte Cloud)
 
-### Requirements
+### Setup guide
+
+Please read [How authenticate with OAuth](https://developers.intercom.com/building-apps/docs/authentication-types#section-o-auth)
```

Comment:
that's not quite right, in airbyte cloud they need to click the auth button. We already set it up for them

Reply:
Updated to review